makefiles/host/rootfs.mk

```diff
@@ -56,3 +59,17 @@ build/nctrl/target/release/nctrl: build/nctrl/.copy_stamp
 	FUSE_CROSS_STATIC_PATH=./thirdparty/ \
 	FUSE_CROSS_STATIC_LIB=fuse \
 	cargo build --release --target=armv7-unknown-linux-musleabihf
+
+$(OPENOCD_SOURCE): build/root.fs/.base_install
+	@mkdir -p $(@D)
+	rm -rf $@
+	git clone https://repo.or.cz/openocd.git $@
+	(cd $@ && git reset --hard $(OPENOCD_VERSION))
+	touch $@/.scmversion
+
+$(OPENOCD_SOURCE)/.build_stamp: $(OPENOCD_SOURCE) build/root.fs/.base_install
+	(cd $(@D) && ./bootstrap)
+	(cd $(@D) && PKG_CONFIG=/root/armv7-eabihf--musl--bleeding-edge-2020.02-2/bin/pkg-config ./configure --host=arm-buildroot-linux-musleabihf --disable-shared --disable-backtrace --enable-static --prefix=$${PWD}/../root.fs/usr/ --enable-sysfsgpio )
```

Comment:
You can probably leave out `--disable-shared` and `--disable-backtrace`, `--disable-shared` is for disabeling shared libraries to be built (but openocd doesn't build any libraries anyways) and `--disable-backtrace` disables the use of `libbacktrace` that openocd doesn't use anyways.

Reply:
okay i see